index.md

```diff
@@ -13,21 +21,35 @@ electronic structure codes, it has become possible to perform large sets
 of calculations automatically. This is the burgeoning area of
 high-throughput ab initio computation. Such calculations have been used
 to create large databases containing the calculated properties of
-existing and hypothetical materials, many of which have appeared online:
-
-- [the AFLOW distributed materials property repository](http://aflowlib.org/)
-- [the Harvard Clean Energy Project Database](http://molecularspace.org/)
-- [the Materials Cloud](http://materialscloud.org/)
-- [the Materials Project](http://materialsproject.org/)
-- [the NoMaD (Novel Materials Discovery) Repository](http://nomad-repository.eu/)
-- [the Open Quantum Materials Database](http://oqmd.org/)
-- [the Computational Materials Repository](http://cmr.fysik.dtu.dk/)
-- [the Data Catalyst Genome](http://suncat.stanford.edu/)
-- [the Materials Platform for Data Science](http://mpds.io/)
-- [the Joint Automated Repository for Various Integrated Simulations](https://jarvis.nist.gov)
-- ...
+existing and hypothetical materials, many of which have appeared online.
+
+We have released version 1.0 of the OPTIMADE specification, with several
+databases already providing implementations. A full list is available on
+the [OPTIMADE providers dashboard](https://www.optimade.org/providers-dashboard/).
```

Comment:
I know it's internal, but it isn't yet a proper part of the website, and essentially functions as a separate website, functionally.
```suggestion
the [OPTIMADE providers dashboard](https://www.optimade.org/providers-dashboard/){:target=_blank}.
```

Reply:
I'm not entirely sure I see the value for all these blank targets, especially in this case. If someone wants to open it in a new tab they can do that very easily? Happy to be persuaded otherwise, but I think for now I will merge as is.

Reply:
One thing that makes this argument simpler: it looks like Jekyll doesn't use the blank target when generating HTML, so we don't actually have any at all in the hosted site :grin:

Reply:
see e.g. https://stackoverflow.com/a/63546509

Reply:
Ah. I see. Just checked - indeed this is so. Well then I'll update the #29 PR accordingly, removing it completely there 👍

Reply:
Ah! I've found that it _does_ work, one just needs to use quotation marks (check out links to `master` and `develop` branches under Specification: https://www.optimade.org/optimade).